app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.order(:name)
+  end
```

Comment:
- [ ] #index action for user controller should render all users to its corresponding index view page. Kindly, implement it as below:
```Ruby
 def index
      @users = User.all
  end

```